api/RTCPeerConnection.json

```diff
@@ -3969,7 +3970,8 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": "22"
+              "version_added": "22",
+              "notes": "Firefox does not currently support descriptions of type <code>pranswer</code>."
```

Comment:
```suggestion
              "notes": "Firefox does not support descriptions of type <code>pranswer</code>."
```

Reply:
Is there a particular reason you suggest removing the word "currently" here? It's something that eventually it may support, so it is a more precise statement with that word.